test/api-test.js

```diff
@@ -698,6 +715,388 @@ describe('RippleAPI', function () {
           'prepare'));
   });
 
+  /**
+   * prepareTransaction() - new in ripple-lib 1.0.0
+   */
+
+  describe('prepareTransaction - Payment', function () {
```

Comment:
these seem like a lot of tests for a function that just calls out to another function. Just want to make sure these aren't duplicated anywhere else that would make this hard to maintain.

Reply:
Good point. I like the peace-of-mind that all of these transaction types and parameters work; `prepareTransaction` already existed, but it wasn't exposed publicly or tested.